substratevm/src/com.oracle.svm.agent/src/com/oracle/svm/agent/BreakpointInterceptor.java

```diff
@@ -621,12 +627,16 @@ private static boolean findResource(JNIEnvironment jni, JNIObjectHandle thread,
         JNIObjectHandle self = getReceiver(thread);
         JNIObjectHandle module = getObjectArgument(thread, 1);
         JNIObjectHandle name = getObjectArgument(thread, 2);
-        JNIObjectHandle result = Support.callObjectMethodLL(jni, self, bp.method, module, name);
-        if (clearException(jni)) {
-            result = nullHandle();
+        Object result = null;
+        if (!trackMissingMetadata) {
```

Comment:
We will not be using `trackMissingMetadata` in the agent. You can remove this code unconditionally.